bootstrap/bootstrap.go

```diff
@@ -219,6 +221,35 @@ func startServer(
 		signer, err = crypto.NewInMemorySigner(cfg.COAKey, crypto.SHA3_256)
 	case cfg.COAKeys != nil:
 		signer, err = requester.NewKeyRotationSigner(cfg.COAKeys, crypto.SHA3_256)
+	case cfg.COACloudKMSKeys != nil:
+		kmsKeys := make([]flowGoKMS.Key, len(cfg.COACloudKMSKeys))
+		for i, key := range cfg.COACloudKMSKeys {
+			// key has the form "{keyID}@{keyVersion}", where
+			keyParts := strings.Split(key, "@")
+			if len(keyParts) != 2 {
+				return fmt.Errorf("wrong format for Cloud KMS key: %s", key)
+			}
+			kmsKeys[i] = flowGoKMS.Key{
+				ProjectID:  cfg.COACloudKMSProjectID,
+				LocationID: cfg.COACloudKMSLocationID,
+				KeyRingID:  cfg.COACloudKMSKeyRingID,
+				KeyID:      keyParts[0],
+				KeyVersion: keyParts[1],
+			}
+		}
```

Comment:
parsing and building of keys shouldn't be here, this should be part of parsing config function (FromFlags). The Config type should only contain the `[]flowGoKMS.Key` type

Reply:
Good point :+1: 
Updated in https://github.com/onflow/flow-evm-gateway/pull/282/commits/9a3e78e3e08b3eb030231b4f4df8fc373bbbbfb5